stable/oauth2-proxy/templates/ingress.yaml

```diff
@@ -2,7 +2,7 @@
 {{- $serviceName := include "oauth2-proxy.fullname" . -}}
 {{- $servicePort := .Values.service.port -}}
 {{- $ingressPath := .Values.ingress.path -}}
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Actually, according to https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/, this appeared in 1.14 and this is way too recent to merge it in this hardcoded way.

Reply:
Good call-out, will revert this particular change.  It should be noted that `extensions/v1beta1` will be removed in kubernetes 1.20 and will need to be addressed at some point.